tests/test_main.py

```diff
@@ -43,27 +50,7 @@ def test_streaming():
     assert num_batches > 1
 
 
-def test_timeout():
-    client = get_test_client()
-    query = """SELECT block_number,
-       TO_TIMESTAMP(block_timestamp) as block_timestamp,
-       avg(gas) as avg_gas_used,
-       avg(max_priority_fee_per_gas) as avg_max_priority_fee_per_gas,
-       avg(gas_price) as avg_gas_price,
-       avg(gas_price / 1e9) AS avg_gas_price_in_gwei,
-       avg(gas * (gas_price / 1e18)) AS avg_fee_in_eth
-FROM eth.transactions
-WHERE block_timestamp > UNIX_TIMESTAMP()-60*60*24*10 -- last 30 days
-GROUP BY block_number, block_timestamp
-ORDER BY block_number DESC"""
-    try:
-        _ = client.query(query, timeout=1)
-        assert False
-    except TimeoutError:
-        assert True
```

Comment:
why did we get rid of this? would be nice to continue testing quality-of-life features around timing and failures modes so we know we're releasing a high-quality api.